ReactViewControl/ReactViewRender.cs

```diff
@@ -1,46 +1,47 @@
 ﻿using System;
-using System.Collections.Concurrent;
 using System.Collections.Generic;
 using System.IO;
 using System.Linq;
 using System.Reflection;
 using System.Security.Cryptography;
 using System.Text;
-using System.Threading.Tasks;
 using System.Web;
 using System.Windows;
 using System.Windows.Controls;
 using WebViewControl;
 
 namespace ReactViewControl {
 
-    internal partial class ReactViewRender : UserControl, IExecutionEngine, IDisposable {
+    internal partial class ReactViewRender : UserControl, IDisposable {
 
-        private const string ModulesObjectName = "__Modules__";
+        private enum LoadStatus {
+            Initialized,
+            PageLoaded,
+            Ready
+        }
+
+        internal const string ModulesObjectName = "__Modules__";
         private const string ComponentLoadedEventName = "ComponentLoaded";
 
-        private static readonly Assembly ResourcesAssembly = typeof(ReactViewResources.Resources).Assembly;
+        private static Assembly ResourcesAssembly { get; } = typeof(ReactViewResources.Resources).Assembly;
```

Comment:
Do you really need this as a property (since the usages have not been changed)?

Reply:
Same for properties below

Reply:
This is a readonly property... using the "new" c# syntax

Reply:
👍